platform/platform-configuration-listener/src/main/java/org/codice/ddf/configuration/store/ConfigurationStatusService.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.configuration.store;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Collection;
+
+public interface ConfigurationStatusService {
+    
+    public Collection<Path> getImportStatus() throws IOException;
```

Comment:
Since this method only returns the list of failed configuration files, should we name it something like `getFailedConfigurationFiles()` or something like that?

I'm also wondering if we should only return Paths or if we should consider returning some kind of status object. Thoughts?